### ISSUE TITLE
I want to get the prob the input two picture?

### ISSUE THREAD

**Issue by mgq1507** [anson0910/Siamese_Networks_Face_Recognition#2]
If I change the loss='binary crossentropy' into loss='softmax',I can get the prob?
If you can give me some advice? I'm appreciate it.
Another, I have trained the network but it costs too much time,how long you train?

**Comment by mgq1507** [comment]
@anson0910

**Comment by anson0910** [comment]
I think you should be able to get the prob similarly by
`prob = self.model.predict([img1, img2])` 

I recall using GTX 970, it took me around 8 hours to train.

**Comment by mgq1507** [comment]
Thank you ,my traing loss curve like this:
![1](https://user-images.githubusercontent.com/32251588/41470453-33c4f6fa-70e3-11e8-935e-ae87dd4f0e51.PNG)
The curve not like yours.
s_net.train(starting_batch=40000),The trained model store where.
In the following code,the test example, not like yours.
Can you tell me how I can recurrent your result?
Thank you again,@anson0910.

**Comment by anson0910** [comment]
I'm not sure why your results are quite different..
Regarding how to save the model, you can check this [link](https://keras.io/getting-started/faq/#how-can-i-save-a-keras-model).

Hope it helps, thanks!
